opcua/ua/uatypes.py

```diff
@@ -341,6 +343,7 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
 
+# FIXME: This is not according to standard, check Part 3, Page 59
```

Comment:
NodeIdTypes are IdentifierTypes if I understood correctly, there are defined in Part 3, Page 59, Table 18 so they are:

* NUMERIC_0
* STRING_1
* GUID_2
* OPAQUE_3

Sorry if I got it wrong.